pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[tool.ruff]
```

Comment:
this was a `ruff` suggestion for [conf/settings](https://github.com/astral-sh/ruff/blob/main/python/ruff/__main__.py) @tuulos